lib/post-json.js

```diff
@@ -22,23 +23,23 @@ function _performLocalDeployment(service, baseURL, what, callback) {
 
   var postURL = url.parse(baseURL);
   if (postURL.protocol !== 'http:') {
-    console.error('Invalid URL `%s`. Only http:// URLs are supported.',
+    glb.error('Invalid URL `%s`. Only http:// URLs are supported.',
       baseURL
     );
-    return callback(Error('Invalid URL'));
+    return callback(glb.Error('Invalid URL'));
   }
 
   if (postURL.hostname) {
     if (postURL.hostname !== 'localhost' && postURL.hostname !== '127.0.0.1') {
-      console.error('Invalid URL `%s`. Only localhost URLs are supported.',
+      glb.error('Invalid URL `%s`. Only {{localhost}} URLs are supported.',
         baseURL);
-      return callback(Error('Invalid URL'));
+      return callback(glb.Error('Invalid URL'));
```

Comment:
```js
return callback(Error(glb.t('Invalid URL')));
```
works too.  The question is if we should remove Error from strong-globalize API.